src/helpers/PageLoader.js

```diff
@@ -1,38 +1,38 @@
 import React from 'react';
```

Comment:
My main idea is to create Magnolia context first from the URL and the rootNodePath. After that, we add more params needed for the SaaS environment like the subscription ID to the context object. Based on the context, we create the URL to make REST calls to fetch the content and annotations.